examples/hal/quad/main.rs

```diff
@@ -146,15 +145,13 @@ fn main() {
     // Setup renderpass and pipeline
     let set_layout = device.create_descriptor_set_layout(&[
             pso::DescriptorSetLayoutBinding {
-                binding: 0,
+                bindings: 0 .. 1,
                 ty: pso::DescriptorType::SampledImage,
-                count: 1,
                 stage_flags: ShaderStageFlags::FRAGMENT,
             },
             pso::DescriptorSetLayoutBinding {
-                binding: 1,
+                bindings: 1 .. 2,
                 ty: pso::DescriptorType::Sampler,
-                count: 1,
```

Comment:
Aren't `binding` index and array `count` orthogonal?

Reply:
I might have lost in the bindings, wish a better type level distinction there